### DISCUSSION TITLE
Relationships

### DISCUSSION THREAD

**Discussion by 4713n** [marcreichel/igdb-laravel#53] [Q&A] [answered]
Hi,
I am trying to get games with cover details, like cover URL etc., but I am not sure how the "Relationships (Extends)" works.

You have this in the examples:
`Game::with(['cover' => ['url', 'image_id']])->get();`

But it returns
`RequestException: HTTP request returned status code 400: [ { "title": "Syntax Error", "status": 400 } ]`

And  when I print query using getQuery() method:
`Game::with(['cover' => ['url', 'image_id']])->getQuery();`
 I get this:
`"fields *,,cover.url,cover.image_id;"`
which doesnt seem to be generated correctly (notice that extra comma).

After removing that extra comma and running query
`fields *,cover.url,cover.image_id;`
in Postman, I get correct results, including cover for each game:
`{
        "id": 164035,
        "artworks": [
            {
                "id": 46305,
                ....
          }
        ],
        "category": 0,
        "cover": {
            "id": 61466,
            "image_id": "yektasaxw6l4ejte7ljb",
            "url": "//images.igdb.com/igdb/image/upload/t_thumb/yektasaxw6l4ejte7ljb.jpg"
        },
        "created_at": 1628766152,
        ...
}`

Or am I doing something wrong?

**Comment by marcreichel** [comment, marked as answer]
Hi, which version of this package are you using?
The bug with the double comma is fixed in version 3.2.1.
So you may update to the latest version in order to fix this issue.
Let me know if this helped.

**Reply by 4713n** [reply]
Thank you very much, I was using older version 2.4.
